src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -4433,51 +4433,57 @@ int32_t NaClSysWaitpid(struct NaClAppThread *natp,
     }
     NaClXCondVarBroadcast(&nap->children_cv);
     NaClXMutexUnlock(&nap->children_mu);
+    ret = pid;
     goto out;
   }
 
   /*
-   * TODO: implement pid == WAIT_ANY_PG (0) and pid == WAIT_ANY (-1) behavior
-   *
-   * -jp
+   * WAIT or WAITPID(-1)
+   * TODO: implement pid == WAIT_ANY_PG (0), we currently don't deal with process groups
```

Comment:
We still have WAIT_ANY (-1) unimplemented, right?

Reply:
If so make a note of that

Reply:
The second part of waitpid() here is what deals with WAIT_ANY, which is the same as a normal wait().